controller/backup_controller.go

```diff
@@ -636,7 +636,19 @@ func (bc *BackupController) checkMonitor(backup *longhorn.Backup, volume *longho
 	if kubernetesStatus.PVCName != "" && kubernetesStatus.LastPVCRefAt == "" {
 		pvc, _ := bc.ds.GetPersistentVolumeClaim(kubernetesStatus.Namespace, kubernetesStatus.PVCName)
 		if pvc != nil {
-			storageClassName = *pvc.Spec.StorageClassName
+			if pvc.Spec.StorageClassName != nil {
+				storageClassName = *pvc.Spec.StorageClassName
+			}
+			if storageClassName == "" {
+				if v, exist := pvc.Annotations["volume.beta.kubernetes.io/storage-class"]; exist {
```

Comment:
use `corev1.BetaStorageClassAnnotation` instead.

Reply:
thanks for reminding the const name!

Reply:
Do you think we need to support alpha
I didn't find alpha any k8s doc
but from users comment
```
---
kind: "PersistentVolumeClaim"
apiVersion: "v1"
metadata:
  name: {{ template "portainer.fullname" . }}
  namespace: {{ .Release.Namespace }}
  annotations:
  {{- if .Values.persistence.storageClass }}
    volume.beta.kubernetes.io/storage-class: {{ .Values.persistence.storageClass | quote }}
  {{- else }}
    volume.alpha.kubernetes.io/storage-class: "generic"
  {{- end }}
  ....
  ....
```
They have alpha ?

Reply:
I'm fine with getting from `alpha`, because the PVC manifest might follow old rules or from created from old cluster.

Reply:
Got it, but for alpha, there is no const defined for it now.

Reply:
Fixed.

Reply:
Yeah, just keep it as it is.

Reply:
There is no `volume.alpha.kubernetes.io/storage-class` or something similar defined in `k8s.io/api/core/v1/types.go`. Do we need to consider it? Or how can we reproduce this...?

Reply:
I have tested the annotation
`volume.alpha.kubernetes.io/storage-class` won't work
the pvc will use the default storageClass in the cluster instead, and k8s will add the `storageClassName` to the pvc.Spec automatically in this case

I think we don't have to handle alpha annotation
cc @derekbit @shuo-wu

Reply:
I am fine with ignoring the alpha annotation as it's probably not used by Kubernetes.

Reply:
Sure. logging it here is fine as well, e.g. `failed to find storageclass from the pvc....`

Reply:
Fixed.